packages/salesforcedx-vscode-core/src/commands/forceCreateManifest.ts

```diff
@@ -38,7 +39,9 @@ export class ManifestCreateExecutor extends LibraryCommandletExecutor<string> {
                    progress?: vscode.Progress<{ message?: string | undefined; increment?: number | undefined; }>,
                    token?: vscode.CancellationToken): Promise<boolean> {
     if (this.sourcePaths) {
+      const sourceApiVersion = (await SfdxProjectConfig.getValue('sourceApiVersion')) as string;
       const componentSet = ComponentSet.fromSource(this.sourcePaths);
+      componentSet.sourceApiVersion = sourceApiVersion;
```

Comment:
👍 Can you ensure that we have a test that covers this change here: `packages/salesforcedx-vscode-core/test/vscode-integration/commands/forceCreateManifest.test.ts`?